src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -2493,6 +2494,20 @@ public void addRestrictedSummary(Element forWhat, Content target) {
 
     public void addPreviewInfo(Element forWhat, Content target) {
         if (utils.isPreviewAPI(forWhat)) {
+            // Preview note tag may be used to provide an alternative preview note.
+            String previewNoteTag = configuration.getOptions().previewNoteTag();
+            if (previewNoteTag != null) {
+                List<? extends UnknownBlockTagTree> tags = utils.getBlockTags(forWhat,
+                        t -> t.getTagName().equals(previewNoteTag), UnknownBlockTagTree.class);
+                if (tags != null && !tags.isEmpty()) {
+                    var previewDiv = HtmlTree.DIV(HtmlStyles.previewBlock);
+                    previewDiv.setId(htmlIds.forPreviewSection(forWhat));
+                    previewDiv.add(HtmlTree.DIV(HtmlStyles.previewComment,
+                            commentTagsToContent(forWhat, tags.getFirst().getContent(), false)));
```

Comment:
Should we add a check to assert there is exactly one occurrence of such a preview note tag?